src/Models/Nonhydrostatic2DModels/nonhydrostatic_2d_model.jl

```diff
@@ -36,13 +35,10 @@ function Models.default_initial_conditions(model::Nonhydrostatic2DModel)
 end
 
 function Models.make_ode_function(model::Nonhydrostatic2DModel)
-    FT = eltype(model.parameters)
-    @unpack p_0, R_d, cp_d, cv_d, g = model.parameters
-    γ = cp_d / cv_d
+    FT = eltype(model.domain)
 
-    # TODO!: Replace with Thermodynamics.jl
-    pressure(ρθ::FT) where {FT} =
-        ρθ >= FT(0) ? p_0 * (R_d * ρθ / p_0)^γ : FT(NaN)
+    # relevant parameters 
+    g::FT = CP.Planet.grav(model.parameters)
```

Comment:
Can we define the const CP and TD in this file, or simply use the full package name? It took me a while to figure out what they are and where they are defined.

Reply:
Ok